scripts/frecency-walk.sh

```
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# frecency-walk.sh — one nav-ring walk for the frecency E2E drive (PR #218).
+# Anchors the executor at $1 (a tab id), then presses {"row":N} for each row
+# 1..$2 and prints the focused tab after each landing. Landings are the
+# drive's machine-readable read-back of rows() order (dump-screen returns
+# nothing for plugin panes — FOOTGUNS).
+set -uo pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)